client/src/utils/slate/closestPointerExportAncestor.ts

```diff
@@ -0,0 +1,15 @@
+export function closestPointerExportAncestor(node: any, document: any) {
+  let curNode = node;
+  while (
+    !(curNode.type && curNode.type === "pointerExport")
+    &&
+    document.getParent(curNode.key)
+  ) {
+    curNode = document.getParent(curNode.key);
+  }
+
+  if (curNode.type && curNode.type === "pointerExport") {
```

Comment:
I personally have liked using ``_.get`` for doing this, it's optional of course.